.pipelines/cg-pipeline.yaml

```diff
@@ -43,4 +43,100 @@ stages:
               fi
             name: "EnvironmentalVariables"
             displayName: "Set environmental variables"
-            condition: always()
\ No newline at end of file
+            condition: always()
+
+      - job: retinaagentimages
+        displayName: Build Retina Agent Images
+        pool:
+          name: "$(BUILD_POOL_NAME_DEFAULT)"
+        strategy:
+          matrix:
+            agent-linux-arm:
+              platform: "linux"
+              arch: "arm64"
+              target: "agent"
+
+            agent-linux-amd64:
+              platform: "linux"
+              arch: "amd64"
+              target: "agent"
+
+            init-linux-arm:
+              platform: "linux"
+              arch: "arm64"
+              target: "init"
+
+            init-linux-amd64:
+              platform: "linux"
+              arch: "amd64"
+              target: "init"
+
+        steps:
+          - checkout: self
+            fetchTags: true
+          - script: |
+              set -euo pipefail
+              echo "VERSION=$(make version)"
+              export VERSION=$(make version)
+              mkdir -p ./output/images/$(platform)/$(arch)/$(year)
+              make retina-image \
+                AGENT_TARGETS=$(target) \
+                TAG=$(make version) \
+                IMAGE_NAMESPACE=retina \
+                PLATFORM=$(platform)/$(arch) \
+                IMAGE_REGISTRY=ghcr.io/microsoft  \
+                BUILDX_ACTION="-o type=docker,dest=./output/images/$(platform)/$(arch)/retina-$(target)-$VERSION.tar"
```

Comment:
can we add os for the images keeping the parity with the gh images. retina-agent-$VERSION-linux-$(arch).tar